src/theme/index.tsx

```diff
@@ -17,7 +17,7 @@ const {
   //   see: https://github.com/palantir/tslint/issues/3505
   //   It's possibly due to the rule upstream in tslint-config-standard
   // tslint:disable-next-line
-} = styledComponents as styledComponents.ThemedStyledComponentsModule<IThemeProps>
+} = styledComponents as styledComponents.ThemedStyledComponentsModule<any>
```

Comment:
There's got to be a better way than just using `any` here, right? Wouldn't this lose all intellisense for theme props in all brave components (`${p => p.theme.xyz}`)? Perhaps `Partial<IThemeProps>` would work?

Reply:
Partial won't work. `<IThemeProps & any>` might, but not sure what that will get us. Maybe it's good. Either way seem comment on https://github.com/brave/brave-core/pull/3300 as to whether this is needed yet.